ww-config.js

```diff
@@ -1,639 +1,751 @@
+        },
+        showAllDayEvents: {
+            label: {
+                en: 'Show all day events',
+            },
+            type: 'OnOff',
+            defaultValue: false,
+            bindable: true,
+        },
+        showCountOnYearView: {
+            label: {
+                en: 'Show count on year view',
+            },
+            type: 'OnOff',
+            defaultValue: false,
+            bindable: true,
+        },
+        enableTimelessMode: {
+            label: {
+                en: 'Timeless mode',
+            },
+            type: 'OnOff',
+            defaultValue: false,
+            bindable: true,
+        },
+        timestep: {
+            hidden: content => content.enableTimelessMode,
+            label: {
+                en: 'Timestep (minutes)',
+            },
+            type: 'Number',
+            options: {
+                min: 5,
+                max: 60,
+                step: 5,
+            },
+            defaultValue: 30,
+            bindable: true,
+        },
+        timeStart: {
+            hidden: content => content.enableTimelessMode,
+            label: {
+                en: 'Time start (hour)',
+            },
+            type: 'Number',
+            options: {
+                min: 0,
+                max: 23,
+                step: 1,
+            },
+            defaultValue: 8,
+            bindable: true,
+        },
+        timeEnd: {
+            hidden: content => content.enableTimelessMode,
+            label: {
+                en: 'Time end (hour)',
+            },
+            type: 'Number',
+            options: {
+                min: 1,
+                max: 24,
+                step: 1,
+            },
+            defaultValue: 20,
+            bindable: true,
+        },
+        startWeekOnSunday: {
+            label: {
+                en: 'Start week on Sunday',
+            },
+            type: 'OnOff',
+            defaultValue: false,
+            bindable: true,
+        },
+        hideWeekdays: {
+            label: {
+                en: 'Hide particular days of the week',
+            },
+            type: 'Array',
+            defaultValue: [],
+            bindable: true,
+            options: {
+                item: {
+                    type: 'TextSelect',
+                    options: {
+                        options: [
+                            { label: 'Monday', value: 1 },
+                            { label: 'Tuesday', value: 2 },
+                            { label: 'Wednesday', value: 3 },
+                            { label: 'Thursday', value: 4 },
+                            { label: 'Friday', value: 5 },
+                            { label: 'Saturday', value: 6 },
+                            { label: 'Sunday', value: 7 },
+                        ],
+                    },
+                },
+            },
+            /* wwEditor:start */
+            bindingValidation: {
+                type: 'string',
```

Comment:
Shouldn't this be array ?

Reply:
Indeed. It does expect a value like `[0, 1, 2, 3, 4, 5]`. I've updated it.

It's hard to test because no matter what value I use in `bindingValidation`, nothing displays on WeWeb.
<img width="771" alt="image" src="https://github.com/weweb-assets/ww-calendar/assets/107790761/4feeb7be-bf14-44a7-84e0-b3fae211e150">